### DISCUSSION TITLE
Changing URLs

### DISCUSSION THREAD

**Discussion by giliomeejg** [tighten/ziggy#718] [Ideas]
It would be great to be able to override the url parameter.

My situation is that I've got a domain.com (CDN) and origin.domain.com (Server), and I want to be able to serve up the application with relative URLs.  While the Ziggy library allows for passing a 2nd, url parameter, it appears that this parameter is lost in BladeRouteGenerator.php (line 14)

**Comment by bakerkretzmar** [comment]
If you just want relative URLs, you can pass `false` as the third argument to Ziggy's `route()` function.
